docs/redis-clients/java.md

```diff
@@ -3,19 +3,19 @@ title: "Java guide"
 linkTitle: "Java"
 description: Connect your Java application to a Redis database
 weight: 3
-​
+
 ---
-​
+
 Install Redis and the Redis client, then connect your Java application to a Redis database. 
-​
+
 ## Jedis
-​
+
 [Jedis](https://github.com/redis/jedis) is a Java client for Redis designed for performance and ease of use.
-​
+
 ### Install
-​
+
 To include `Jedis` as a dependency in your application, edit the `pom.xml` dependency file, as follows.

```

Comment:
@nermiller `pom.xml` exists only in Maven projects, so the line 

> edit the `pom.xml` dependency file

is only relevant for the Maven bullet point.